src/core/simulation.jl

```diff
@@ -1287,6 +1281,14 @@ struct SimulationSerializationWrapper
     name::String
 end
 
+function _empty_problem_caches!(sim::Simulation)
```

Comment:
can you specify this is about time series cache? since we have the simulation cache it can be confusing

Reply:
I actually made the name generic in case we ever wanted to empty other caches in the same place. If this doesn't change your mind then I'll change the name.